.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: '[Feature Request]'
+labels: ['enhancement', 'needs-triage']
+assignees: ''
+
+---
+
+<!--- If this is a language feature request please file the issue against the
+hlsl-specs repository: https://github.com/microsoft/hlsl-specs/issues/new --->
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
I think we should push more for the detailed description of the problem itself. I would remove the question and say something like "Describe the problem in as much detail that led you to request this feature". 

I don't really want concise. Very often feature requests that we just can't do can be solved another way if we understood the problem.